backend/src/api/bitcoin/bitcoin.routes.ts

```diff
@@ -90,7 +90,7 @@ class BitcoinRoutes {
       .get(config.MEMPOOL.API_URL_PREFIX + 'block/:hash/summary', this.getStrippedBlockTransactions);
       ;
 
-      if (config.MEMPOOL.BACKEND !== 'esplora') {
+      if (config.MEMPOOL.BACKEND !== 'none') {
```

Comment:
We also have a 3rd backend option other than "esplora and "none" (Bitcoin Core only) called "electrum" when running the more limited `romanz/electrs` or Fulcrum where these API endpoints doesn't exist, so the check to enable them must be `esplora` only.